app/src/main/java/com/zulip/android/networking/AsyncGetEvents.java

```diff
@@ -190,7 +190,13 @@ public Void call() throws Exception {
                             } else {
                                 personDao.update(person);
                             }
-                        } catch (Exception e) {
+                            if (person.getEmail().equals(app.getEmail())) {
```

Comment:
This is also required as when we refresh (from menu) name in `app.getYou()` becomes null. To update name this is required.